scripts/livepeer-nginx

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [[ $* == *-j* ]]; then
+    echo '{
+  "desc": "Livepeer-packaged nginx",
+  "name": "Livepeer nginx",
+  "optional": {}
+}'
+    exit 0
+fi
+
+cat > /etc/nginx/nginx.conf << EOF
```

Comment:
It could be placed it in the `scripts/nginx.conf` file.. and then `cp ./nginx.conf /etc/nginx/nginx.conf`

Reply:
That's true, though then we'd be shipping `nginx.conf` in the `bin` directory we export... not sure.

Reply:
I thought about code readability... having `nginx.conf` inside `bin/` on livepeer-in-a-box isn't that incorrect.